### DISCUSSION TITLE
Go through judgement fonts

### DISCUSSION THREAD

**Discussion by ListenerJubatus** [TeamRizu/OutFox#276] [02 - Bug or Troubleshooting Discussions]
Someone reported inconsistencies in the glow on some of the glow fonts, so I might as well just go through and fine tune all of them.

**Comment by lomdr** [comment]
It was me that said something on that.  There's also some offness with the non-offset judgments too (Ace Shaped ITG has late Fantastic a couple of pixels off to the right compared to early Fantastic)
